istioctl/cmd/kubeinject.go

```diff
@@ -474,6 +517,8 @@ It's best to do kube-inject when the resource is initially created.
 			}
 			var reader io.Reader
 
+			l := clog.NewConsoleLogger(c.OutOrStdout(), c.ErrOrStderr(), scope)
```

Comment:
logging anything seems like it will break `kube-inject | kubectl apply` ?

Reply:
Removed since `GenerateConfig ` is not used anymore.